docs/geps/gep-01.rst

```diff
@@ -108,6 +120,10 @@ changed, even if it leads to long variable names (e.g., ``kinderfreib``,
 ``_zu_verst_eink_kein_kinderfreib_tu``). This makes searching for identifiers
 easier and less error-prone.
 
+If names need to be concatenated for making clear what a column name refers to (e.g.,
```

Comment:
Repetition of third bullet above.

Reply:
Yes, on purpose. Some people might read closer up there, some down here. I did not see a way to shortening it in the tl;dr bit nor a point in expanding it down here.

Reply:
alright